src/utils/XhrNetworkInterceptor.ts

```diff
@@ -24,11 +24,11 @@ export interface NetworkData {
   gqlQueryName?: string;
   serverErrorMessage: string;
   requestContentType: string;
-  w3cc: boolean | null;
-  partialId: number | null;
-  etst: number | null;
-  wgeti: string | null;
-  wceti: string | null;
+  isW3cHeaderFound: boolean;
+  partialId: number;
+  networkStartTimeInSeconds: number;
+  W3CgeneratedHeader: string;
+  W3CCaughtHeader: string;
```

Comment:
Let's use `camelCase` for these.
```suggestion
  w3cGeneratedHeader: string;
  w3cCaughtHeader: string;
```